server/chat/background/task.py

```
@@ -498,6 +489,16 @@ def run_background_chat(
     try:
         # Link session and Celery task ID to incident if provided
         is_action_source = (trigger_metadata or {}).get('source') == 'action'
+
+        # Send action started notification (not gated by send_notifications —
+        # that flag controls investigation-level notifications, not action-specific ones)
+        if is_action_source:
+            try:
+                from utils.notifications.dispatcher import notify_action_started
+                notify_action_started(user_id, trigger_metadata, session_id)
+            except Exception as e:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)